### DISCUSSION TITLE
How to install apps not in store for AIO instance.

### DISCUSSION THREAD

**Discussion by brunoalfred** [nextcloud/all-in-one#1287] [Questions] [answered]
While using AIO to setup the NC instance ie one doesn't have access to nextcloud/apps folder, what is the way to install an app that  doesn't exist in app store without placing it in apps directory manually?

**Comment by szaimen** [comment, marked as answer]
Indeed you need to place apps that are not available in the appstore into the apps directory directly as with any other Nextcloud instance.

It should by default be stored in `/var/lib/docker/volumes/nextcloud_aio_nextcloud/_data/custom_apps/`

**Reply by Zoey2936** [reply]
See: https://github.com/nextcloud/all-in-one#how-to-run-occ-commands

**Reply by brunoalfred** [reply]
Seen it. Thanks. Reading the Docs first is important 😃
